bgpd/bgp_aspath.c

```diff
@@ -1299,12 +1300,15 @@ struct aspath *aspath_remove_private_asns(struct aspath *aspath, as_t peer_asn)
 	while (seg) {
 	      public
 		= 0;
+		peer = 0;
 		for (i = 0; i < seg->length; i++) {
 			// ASN is public
-			if (!BGP_AS_IS_PRIVATE(seg->as[i])) {
+			if (!BGP_AS_IS_PRIVATE(seg->as[i]))
 			      public
-				++;
-			}
+			++;
```

Comment:
Could you move this `++` to the proper place? This looks very weird.

Reply:
That came from clang-format directly (I'm guessing it's treating `public` special).  Happy to fix it up manually, but I suspect that will make CI whine about the style.

Let me know how you'd like me to proceed.

Reply:
I think Igor's right - whatever clang says, this is too awkward to keep. not the only pattern that clang gets wrong...

Reply:
Fine by me!  I'll fix this when I resubmit with a topotest :-)